Makefile

```diff
@@ -223,7 +223,7 @@ proto: docker-build-proto
 	./etc/proto/build.sh
 
 # Run all the tests. Note! This is no longer the test entrypoint for travis
-test: clean-launch launch-dev lint enterprise-code-checkin-test docker-build test-pfs-server test-cmds test-libs test-auth test-identity test-license test-enterprise test-worker test-admin test-pps
+test: clean-launch launch-dev lint enterprise-code-checkin-test docker-build test-pfs-server test-cmds test-libs test-auth test-license test-enterprise test-worker test-admin test-pps
```

Comment:
I'm missing something, but where do identity-test get called now?

Reply:
So I'm not sure if this is the best solution, but they aren't getting called anymore because they have no integration tests to run.

It looks like we get an error like this in Circle when we attempt to call it. It is strange because I figured it would just be a no-op.

https://app.circleci.com/pipelines/github/pachyderm/pachyderm/15250/workflows/a1dae3ed-4da9-4645-937b-11de8dd4111c/jobs/233829